code/scripts/articleView.js

```diff
@@ -0,0 +1,25 @@
+//var articleView = {};
+
+//articleView.handleMainNav = function () {
+  //$('.main-nav').on('click', '.tab', function() {
+    //$('.tab-content').hide();
+    //$('#' + $(this).attr('data-content')).fadeIn();
+  //});
+  //$('.main-nav .tab:first').click();
+//};
+
+//articleView.handleMainNav();
+
+
+//$('.tab').click(function() {
+  //console.log('tab clicked');
+//});
+//$('.tab:first').click();
```

Comment:
Pertaining to the about section, you can remove the commented-out code here, but you can include the click event that is on line 17 and place that in your code below to ensure the about section is hidden on initial page load.